multigeiger/multigeiger.ino

```diff
@@ -465,10 +465,10 @@ void loop()
         digitalWrite (PIN_SPEAKER_OUTPUT_N, LOW);
         delayMicroseconds(500);
       }
-    }
-
-    if(ledTick & !speakerTick) {
-      delay(4);
+    } else {
+      if(ledTick && sw[LED_ON]) {
```

Comment:
this now includes the LED switch check and is consistent with the other 2 checks that turn the pin on/off.